scripts/schema/config/solver.json

```diff
@@ -119,7 +118,8 @@
         "SuperLU3D": { "type": "boolean" },
         "AMSVector": { "type": "boolean" },
         "DivFreeTol": { "type": "number", "minimum": 0.0 },
-        "DivFreeMaxIts": { "type": "integer", "minimum": 0 }
+        "DivFreeMaxIts": { "type": "integer", "minimum": 0 },
+        "GSOrthogonalization": { "type": "string" }
```

Comment:
`, "enum": ["MGS", "CGS", "CGS2"]}` might as well reward those who use the json schema with an earlier failure if they make a typo in their config. Very 80:20 in terms of impact so enum upgrading should probably be done incrementally when new fields are added or modified, or as we find fields that people regularly make typos/errors on rather than for everything rather than a whole sale upgrade which would be a lot of labour while the gain is unknown.

Reply:
I think it should be a separate PR to update the entire JSON Schema to use enums as appropriate. At least in the entire `sovler.json` for consistency.